### DISCUSSION TITLE
Creating GIFs of slider's outcome as animation

### DISCUSSION THREAD

**Discussion by naveenatmit** [fonsp/Pluto.jl#2663] [Questions about using Pluto]
Is there a way to record sliding outcomes as GIF built-in in Pluto? 
What's the best way to create GIFs in general? 

The drawback of html videos is that it is not embeddable inside a google slide!
How to communicate the sliding effects to audience through normal documentation tools like doc, slide, sheet, etc.,?

**Comment by fonsp** [comment]
Hey @naveenatmit!

I think the best way is to record the GIF with a separate program. On Windows I recommend ScreenToGif, on Mac you could use the built in video recorder, and "gifski" to create the GIF.
